### PR TITLE
feat: add scroller component for Dropdown

### PR DESCRIPTION
**Pull request by im36-123**
Currently, Dropdown component cannot mix scrollable area and fixed area. This PR solve that problem.

I added a `DropdownScrollArea` component to distinguish between scrollable areas and fixed areas.

BREAKING CHANGE: The scroll area is no longer automatically allocated.

### CODE REVIEW

File: src/components/Dropdown/DropdownContent.tsx [hunk 1]
```diff
@@ -28,13 +31,14 @@ export const DropdownContent: React.FC<Props> = ({
...
     <DropdownContentRoot>
-      <DropdownContentContext.Provider value={{ onClickCloser }}>
+      <DropdownContentContext.Provider value={{ onClickCloser, controllable, scrollable }}>
         <DropdownContentInner
           triggerRect={triggerRect}
           scrollable={scrollable}
           className={`${dropdownKey} ${className}`}
+          controllable={controllable}
         >
-          {controllable ? children : <DropdownCloser>{children}</DropdownCloser>}
```
Comment (im36-123) [inline src/components/Dropdown/DropdownContent.tsx:37]: I had to pass a value from DropdownContentInner. So we move this logic to DropdownContentInner

File: src/components/Dropdown/DropdownContentInner.tsx [hunk 2]
```diff
@@ -61,15 +71,27 @@ export const DropdownContentInner: FC<Props> = ({
...
       className={`${className} ${isActive ? 'active' : ''}`}
+      controllable={controllable}
       themes={theme}
     >
-      {children}
+      {controllable ? (
+        children
+      ) : (
+        <DropdownContentInnerContext.Provider value={{ maxHeight: contentBox.maxHeight }}>
+          <DropdownCloser>{children}</DropdownCloser>
```
Comment (nabeliwo) [inline src/components/Dropdown/DropdownContentInner.tsx:81]: [IMO] I thought that passing maxHeight as props would not increase the context.

Comment (im36-123) [reply src/components/Dropdown/DropdownContentInner.tsx:81]: Yes, that's right. But doesn't it expose the maxHeight props? Then the user can pass any maxHeight value to DropdownCloser.

Comment (nabeliwo) [reply src/components/Dropdown/DropdownContentInner.tsx:81]: If you don't want the user to touch maxHeight, it certainly looks good 👍

Comment (im36-123) [reply src/components/Dropdown/DropdownContentInner.tsx:81]: It is not expected that the user will pass arbitrary values. I'll improve the components to accept arbitrary values if requested by the user 😄

**Comment by im36-123** [comment]
I will merge this branch if pass the CI check.

**Pull request merged**